### PR TITLE
use interfaces instead of types

### PR DESCRIPTION
**Pull request by Ferotiq**
### Description

Closes #

### CODE REVIEW

**Comment by Bruception** [comment]
I don’t know if this is necessary. Should perhaps be done on a case by case basis where we need to extend types.

**Comment by Ferotiq** [comment]
def not necessary but it's good for consistency and it's because it's what interfaces are made for ¯\_(ツ)_/¯

File: frontend/src/ts/test/custom-text.ts [hunk 1]
```diff
@@ -39,7 +39,9 @@ export function setDelimiter(val: string): void {
   delimiter = val;
 }
 
-type CustomTextObject = { [key: string]: string };
+interface CustomTextObject {
```
Comment (Bruception) [inline frontend/src/ts/test/custom-text.ts:42]: Might as well change this to:

type CustomTextObject = Record<string, string>;

Comment (Miodec) [reply frontend/src/ts/test/custom-text.ts]: @Ferotiq don't forget to comment and resolve conversations if you addressed them.

**Pull request merged**